content/en/blog/releases/Kitex/release-v0_12_0.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Kitex Release v0.12.0"
+linkTitle: "Release v0.12.0"
+projects: ["Kitex"]
+date: 2025-01-13
+description: >
+---
+
+## **Introduction to Key Changes**
```

Comment:
Use H1 title?

Reply:
old one is H2, it is ok

Reply:
I thought the same thing, but every release docs somehow used H2, so I aligned with them.